spec/unit/plugins/vmware_spec.rb

```diff
@@ -0,0 +1,64 @@
+#
+# Author:: "Christopher M. Luciano" <cmlucian@us.ibm.com>
+# Copyright (C) 2015 IBM Corp.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+require 'spec_helper'
+
+describe Ohai::System, "plugin vmware" do
+
+  let(:plugin) { get_plugin("vmware") }
+  let(:path) { "/usr/bin/vmware-toolbox-cmd" }
+  let(:param) { nil }
+
+  context "vmware toolbox" do
+
+    def setup_stubs
+      allow(File).to receive(:exist?).and_return(true)
+      allow(plugin).to receive(:collect_os).and_return(:linux)
+      allow(plugin).to receive(:shell_out).with("/usr/bin/vmware-toolbox-cmd stat speed").and_return(mock_shell_out(0, "2000 MHz", nil))
```

Comment:
You can make use of the `path` variable, which will make stubbing for windows easier in the future:

```ruby
allow(plugin).to receive(:shell_out).with("#{path} stat hosttime").and return(...)
```